testdata/project-v3-addon/PROJECT

```diff
@@ -3,15 +3,21 @@ layout: go.kubebuilder.io/v3-alpha
 projectName: project-v3-addon
 repo: sigs.k8s.io/kubebuilder/testdata/project-v3-addon
 resources:
```

Comment:
With a change like this, I wonder if its now worth renaming this field to `api` to match `create api --resource --controller`, which is effectively saying "create an api for this GVK, wiring resource types and a controller if specified":

```yaml
apis:
- resource: true
  controller: true
  gvk:
   group: crew
   version: v1
   kind: Captain
  ...
```

Relates to https://github.com/kubernetes-sigs/kubebuilder/issues/1772#issuecomment-723318555.

Reply:
Hi @estroz, 

Following the concerns regards the suggestion: 

- The create api creates API and not the resource. See that we can have a controller without an api.  
```
apis:
- resource: true
```

- The api and controllers are for the GKV used so we cannot does as suggested. See that each GKV has a controller and a resource related. 

```
- resource: true
  controller: true
  gvk:
 -  group: crew
 -  version: v1
 -  kind: Captain
```
 
IMO the scope of this PR is to store the data. Then, re-design it shows out of the scope, however, I am 100% all in to discuss if we can improve it in a follow-up. 

c/c @Adirio

Reply:
I don't dislike changing the name and I think that API may fit. API = [Resource] + [Controller] + [Webhooks] (all optional).

Reply:
Hi folks, 

By extending Kubernetes API we can create our own apis whcih is the CRDs. A controller is NOT an API at all. Then, because of this I do not agree with the rename of resources per apis. However, it shows not part of the scope of this PR as well. IMO a redesign or new nomenclature proposals would better fit as a new issue for we discuss that and can be applied as a follow-up.

Reply:
>IMO the scope of this PR

While I agree, I also think we've been changing the config file 3-alpha spec a _lot_ recently without a concerted view of these changes, which worries me because we may be fragmenting UX and treating `PROJECT` as something the user should never look at (which they will probably have to at some point). Personally I'd like to see a draft EP for #1826 before adding these fields, one that discusses a general methodology for adding/modifying the existing `resources` field with types created by `create` subcommands.

Reply:
I can work on in an EP as follow up for we are able to address the suggestion made so far. 👍

Reply:
Pushed an doc for the purposed changes + to doc the suggestion made here to add the new attr gkv. See: https://github.com/kubernetes-sigs/kubebuilder/pull/1861/files

Reply:
https://github.com/kubernetes-sigs/kubebuilder/issues/1826#issuecomment-732460408

Reply:
Hi @estroz,

See the comment: https://github.com/kubernetes-sigs/kubebuilder/pull/1849#issuecomment-734230778